dace/frontend/python/replacements.py

```diff
@@ -1718,19 +1827,520 @@ def _matmult(visitor, sdfg: SDFG, state: SDFGState, op1: str, op2: str):
+        name="_numpy_isinf_",
+        operator="FpBoolean",
+        inputs=["__in1"],
+        outputs=["__out"], code="__out = isinf(__in1)",
+        reduce=None, initial=np.isinf.identity),
+    isnan = dict(
+        name="_numpy_isnan_",
+        operator="FpBoolean",
+        inputs=["__in1"],
+        outputs=["__out"], code="__out = isnan(__in1)",
+        reduce=None, initial=np.isnan.identity),
+    signbit = dict(
+        name="_numpy_signbit_",
+        operator="SignBit",
+        inputs=["__in1"],
+        outputs=["__out"], code="__out = signbit(__in1)",
+        reduce=None, initial=np.signbit.identity),
+    copysign = dict(
+        name="_numpy_copysign_",
+        operator="CopySign",
+        inputs=["__in1", "__in2"],
+        outputs=["__out"], code="__out = copysign(__in1, __in2)",
+        reduce="lambda a, b: copysign(a, b)", initial=np.copysign.identity),
+    nextafter = dict(
+        name="_numpy_nextafter_",
+        operator="NextAfter",
+        inputs=["__in1", "__in2"],
+        outputs=["__out"], code="__out = nextafter(__in1, __in2)",
+        reduce="lambda a, b: nextafter(a, b)", initial=np.nextafter.identity),
+    spacing = dict(
+        name="_numpy_spacing_",
+        operator="Spacing",
+        inputs=["__in1"],
+        outputs=["__out"], code="__out = nextafter(__in1, inf) - __in1",
+        reduce=None, initial=np.spacing.identity),
+    modf = dict(
+        name="_numpy_modf_",
+        operator="Modf",
+        inputs=["__in1"],
+        outputs=["__out1", "__out2"],
+        code="np_modf(__in1, __out1, __out2)",
+        reduce=None, initial=np.modf.identity),
+    ldexp = dict(
+        name="_numpy_ldexp_",
+        operator="Ldexp",
+        inputs=["__in1", "__in2"],
+        outputs=["__out"], code="__out = ldexp(__in1, __in2)",
+        # NumPy apparently has np.ldexp.reduce, but for any kind of input array
+        # it returns "TypeError: No loop matching the specified signature and
+        # casting was found for ufunc ldexp". Considering that the method
+        # computes __in1 * 2 ** __in2, it is hard to define a reduction.
+        reduce=None, initial=np.ldexp.identity),
+    frexp = dict(
+        name="_numpy_frexp_",
+        operator="Frexp",
+        inputs=["__in1"],
+        outputs=["__out1", "__out2"],
+        code="np_frexp(__in1, __out1, __out2)",
+        reduce=None, initial=np.frexp.identity),
+    floor = dict(
+        name="_numpy_floor_",
+        operator="Floor",
+        inputs=["__in1"],
+        outputs=["__out"], code="__out = floor(__in1)",
+        reduce=None, initial=np.floor.identity),
+    ceil = dict(
+        name="_numpy_ceil_",
+        operator="Ceil",
+        inputs=["__in1"],
+        outputs=["__out"], code="__out = ceil(__in1)",
+        reduce=None, initial=np.ceil.identity),
+    trunc = dict(
+        name="_numpy_trunc_",
+        operator="Trunc",
+        inputs=["__in1"],
+        outputs=["__out"], code="__out = trunc(__in1)",
```

Comment:
I agree that the file needs reorganization, but there are a lot of things to consider. Therefore, I don't want to spend time doing a partial (and not final) reorganization. Refactoring the whole thing (e.g. linking python operators to numpy ufuncs, moving the operator/ufunc framework to a library node, removing obsolete function replacements, etc.) would take a lot of time. At the same time, I don't get any feedback on all the new features, because they are not in master. This has to be done in steps.